README.md

```diff
@@ -66,15 +66,20 @@ The [Lighthouse](https://github.com/submariner-io/lighthouse) project helps in c
 
 To deploy Submariner with Lighthouse follow the below steps.
 
+* Create a kubeconfig file which has the context of all the clusters. If  KUBECONFIG environment variable set with the path to all clusters, you could use the below command.
```

Comment:
nit: doublespace between if and KUBECONFG.
nit: "environment variable is set"

Reply:
Done.